Corona-Warn-App/src/main/java/de/rki/coronawarnapp/exception/reporting/ErrorReportReceiver.kt

```diff
@@ -62,26 +63,17 @@ class ErrorReportReceiver(private val activity: Activity) : BroadcastReceiver()
             Timber.v("Not displaying error dialog, not in foreground.")
             return
         }
-
-        val dialogInstance = DialogHelper.DialogInstance(
-            context = activity,
-            title = dialogTitle,
-            message = message,
-            positiveButton = confirm,
-            negativeButton = details,
-            cancelable = null,
-            positiveButtonFunction = {},
-            negativeButtonFunction = {
-                val stackTraceDialog = DialogHelper.DialogInstance(
-                    activity,
-                    title,
-                    "$detailsTitle:\n$stack",
-                    confirm
-                )
-                DialogHelper.showDialog(stackTraceDialog.copy(isTextSelectable = true))
-                Unit
+        (activity as AppCompatActivity).createDialog {
```

Comment:
what we need to from the activity is just the context, can we make the extension function for Activity directly 
` fun Activity.createDialog()`

Reply:
I tried doing this but I need a fragment manager and I'm not sure how to obtain one for the `Activity`. It says it's deppracated.

Reply:
Ah, ok